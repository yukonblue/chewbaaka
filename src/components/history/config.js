/**
 * history/config.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 07, 2020
 * Updated  : Aug 25, 2021
 */

export const config = {
  headMeta: {
    title: "History of the cheetah | Learn about cheetahs",
    description: "Learn about the history and evolution of the cheetah.",
    keywords: [
      "cheetah history",
      "cheetah evolution",
      "cheetah lineage",
      "cheetah population",
      "cheetah range",
    ]
  },
  pageProps: {
    coverImageNamePrefix: "History_Page_Cover_Image",
    title: "History",
    subtitle: "Learn about the evolution and migration of the species, its relationship with man throughout history, and its population and ranges.",
    pageMenuItems: [
      {
        title: "Evolution",
        tocImageFilename: "history_page_toc_menu_img_01-min.jpg"
      },
      {
        title: "Cheetah and Man",
        tocImageFilename: "history_page_toc_menu_img_02-min.jpg"
      },
      {
        title: "Range and Population",
        tocImageFilename: "history_page_toc_menu_img_03-min.jpg"
      }
    ],
    pageTailNavMenu: {
      prevPage: {
        label: "Home",
        href: "/"
      },
      nextPage: {
        label: "Biology",
        href: "/biology/"
      }
    }
  },
  contentPageIntro: {
    title: "History of the Cheetah",
    content: "The evolution and history of the cheetah are just as remarkable and " +
          "interesting as the species itself. Its evolution goes back to nearly " +
          "7 million years ago in time, and spanned across almost all the continents on the planet. " +
          "Its history with human interactions goes as far back as to 32000 BCE, and " +
          "the cheetah has been a symbol of significant meaning in different civilizations and " +
          "cultures across thousands of years of human history. Even up to this day, " +
          "the cheetah continues to be considered as a spiritual animal that manifests " +
          "as a symbol of a multitude of positive characteristics, such as adaptability, " +
          "persistence, and focus.",
    image: {
      filenamePrefix: "History_Page_Intro_Section_Background",
      aspectRatio: {
        width: 861,
        height: 420
      }
    }
  },
  contentPageSections: {
    section_Evolution: {
      intro: {
        title: "Evolution",
        content: "Although many would think the cheetah had originated on the continent of Africa, " +
        "the oldest fossils place  cheetahs in North America in what is now Texas, Nevada and Wyoming. " +
        "Cheetahs were common throughout Asia, Africa, Europe and North America until the end of the last " +
        "Ice Age, about 10,000 years ago, when massive climatic changes caused large numbers of mammals to disappear."
      },
      subsections: {
        subsection_EvolutionOfCats: {
          title: "Evolution of Cats"
        },
        subsection_CheetahEvolution: {
          title: "Cheetah Evolution"
        }
      },
    },
    section_CheetahAndMan: {
      intro: {
        title: "Cheetah and Man",
        content: "The history of cheetah-human relationship goes back in millennia, " +
        "and archeological artifacts uncovered throughout the world suggest that " +
        "human interactions with the species goes hand in hand with its migration and evolution. " +
        "The relationship between cheetahs and us have been both profound and complex, " +
        "as the species have been revered, utilized, displayed, and exploited throughout human history."
      },
      subsections: {
        subsection_RelationshipsWithMan: {
          title: "Relationships with Man",
          contents: {
            paragraph_Cheetah_And_Man_01:
              'Traditional African healers and witchdoctors used cheetah foot bones in spiritualistic rituals to symbolize fleet-footedness and speed. ' +
              'They used bones from a wide variety of animals and techniques varied from tribe to tribe. Objects ' +
              'represented a person, a thing or mood from the past, present or future, and are known as Divinatory sets.',
            paragraph_Cheetah_And_Man_02:
              'During the time of the Egyptian pharaohs, the cheetah was considered a goddess named ' +
              '"Mafdet". Pharaohs kept cheetahs as close companions, ' +
              "symbolic of Mafdet's protection of the royal throne.",
            paragraph_Cheetah_And_Man_03:
              'The San of Southern Africa ate cheetah meat as a symbol for speed, but it was not a main food in their diet.',
            paragraph_Cheetah_And_Man_04:
              'Kings wore cheetah skins for dignity. Trade in cheetah skin only started after European explorers began ' +
              'requesting them.',
          }
        },
        subsection_CheetahsInSports: {
          title: "Cheetahs In Sports",
          contents: {
            paragraph_The_Hunting_Leopard_01:
              'In c1700 BC, the Egyptians were the first to tame cheetahs. They admired the cheetah for its speed, ' +
              'hunting ability and beauty. They honored cheetahs as symbols of royalty and prestige.',

            paragraph_The_Hunting_Leopard_02:
              'The swiftest animal on earth became a cherished hunting companion of Pharaohs and royalty ' +
              'throughout Europe, Asia and India. Until the early 1900s, ownership of cheetahs was as important ' +
              'to these nobles as their love for gold. Cheetahs hunt by sight so they excelled in the sport ' +
              'known as "coursing". Hunts organized by royalty and nobles were for the challenge of sport, ' +
              'not for food. Hunts represented power and prestige.',

            paragraph_The_Hunting_Leopard_03:
              'By the 1500s the popularity of the cheetah as a hunting companion rivaled that of the dog. ' +
              'Cheetahs, the most easily tamed of the big cats, were caught, tamed and trained. Adults were ' +
              'used because cubs had not learned how to hunt. Tamed cheetahs formed a strong bond with their keepers.',

            paragraph_The_Hunting_Leopard_04:
              'Each cheetah rode to the hunt by horseback or on a cart. Its eyes were coved with a hood and uncovered ' +
              'when prey were sighted. The cat was released to chase down the prey then rewarded with meat fed from a ' +
              'wooden spoon.',

            paragraph_The_Hunting_Leopard_05:
              'Although cherished, pampered cheetahs were loved to near extinction and taken from the wild in great numbers. ' +
              'By the early 1900s, India and Iran were importing African cheetahs for the sport of "coursing", as their own ' +
              'wild population became too small.'
          }
        },
        subsection_CheetahsInArt: {
          title: "Cheetahs In Art",
          contents: {
            paragraph_Cheetah_In_Art_01:
              'Sumerians (Iraqi ancestors) were the first to use cheetahs in art.',

            paragraph_Cheetah_In_Art_02:
              'The cheetah and leopard have often been mistaken for each other. ' +
              'Cheetahs in early art were frequently called "panther" ' +
              'or "hunting leopard". Early artists called the true leopard "pardus" ' +
              'and described it as a cross between a cheetah (panther), and the lion (leo).',

            paragraph_Cheetah_In_Art_03:
              'Middle Age and Renaissance artists began drawing cheetahs and other ' +
              'animals more life-like. During this time of scientific exploration, ' +
              'detailed descriptions of animals provided incentive for reality in art. ' +
              'Artists included cheetahs more often in their paintings as the trade in ' +
              'live animals increased.',

            paragraph_Cheetah_In_Art_04:
              'Artists always pictured cheetahs as animals of speed and royalty.'
          }
        }
      }
    },
    section_PopulationAndRange: {
      intro: {
        title: "Range and Population",
        content: "A combination of factors such as game hunting, retaliatory killing, " +
                "habitat encroachment have caused the sharp decline of the global cheetah " +
                "population over the last century, and driven the species out of much of their " +
                "historical range. One of the last strongholds of the species today remain in " +
                "southern Africa, spanning over Namibia and Botswana, which together " +
                "holds two thirds of the species' population."
      },
      subsections: {
        subsection_Namibia: {
          title: "Namibia - Cheetah Capital of the World",
          contents: {
            Cheetah_Capital_Of_The_World: {
              title: "Namibia",
              subtitle: "Cheetah Capital of the World",
              content: {
                paragraph_Cheetah_Capital_Of_The_World_01:
                  "Namibia has the world's largest cheetah population. " +
                  "Approximately 3000 cheetahs share the land with humans, " +
                  "livestock, and wildlife.",

                paragraph_Cheetah_Capital_Of_The_World_02:
                  "Today, the status of the Namibian cheetah is stabilizing. " +
                  "During the 1980s, the population of Namibian cheetahs " +
                  "declined by half. In the 10-year period, nearly " +
                  "7,000 cheetahs were removed from the wild.",

                paragraph_Cheetah_Capital_Of_The_World_03:
                  "In most countries where cheetahs live, their numbers " +
                  "have been reduced to critical levels. The cheetah's " +
                  "survival worldwide is in human hands."
              }
            },
            Namibia_Home_For_The_Future: {
              title: "Namibia",
              subtitle: "Hope for the Future",
              content: {
                paragraph_01:
                  "The greatest hope for the cheetah's survival lies in the " +
                  'pristine countryside of Namibia. Nearly 1000 Namibian farmers ' +
                  "control the fate of the country's cheetahs and the land they live."
              }
            }
          }
        },
        subsection_RoadToExtinction: {
          title: "The Road to Extinction",
          contents: {
            The_Road_To_Extinction: {
              title: "The Road to Extinction?",
              subtitle: "1990",
              content: {
                paragraph_The_Road_To_Extinction_01:
                  "World development, industrialization, automobiles, and airplanes...",

                paragraph_The_Road_To_Extinction_02:
                  "Through the 1900s, man's inventions seemed limitless, yet there is " +
                  "nothing man-made that rivals the speed and efficiency of the cheetah.",

                paragraph_The_Road_To_Extinction_03:
                  "Scientists classified the cheetah as Acinonyx jubatus, yet nobles still " +
                  'referred to it as the "hunting leopard". By the end of the 1800s, cheetahs ' +
                  'were a rarity in Asia Minor and Arabia because of their use in the sport ' +
                  'of "coursing".',

                paragraph_The_Road_To_Extinction_04:
                  'Although it appeared that the cheetahs had a large range, their numbers ' +
                  'within that range existed in small pockets. Cheetahs, farmers and their livestock ' +
                  'all preferred open grasslands for their habitat. Increasing agricultural development ' +
                  'and new settlements played havoc with remaining cheetah populations.',
              }
            },
            Where_Did_The_Cheetah_Go: {
              title: "Where Did the Cheetah Go?",
              subtitle: "1975",
              content: {
                paragraph_Where_Did_The_Cheetah_Go_01:
                  'As human population grew, the amount of land devoted to livestock farming ' +
                  'steadily increased. Livestock filled the open land where cheetahs roamed. ' +
                  'Natural prey became scarce. Farmers kill other large predators. Although ' +
                  'game reserves protected them, cheetahs could not compete against hyenas ' +
                  'and lions. Farmlands offered cheetahs a safe haven, but they sometimes killed livestock.',

                paragraph_Where_Did_The_Cheetah_Go_02:
                  'Farmers saw cheetahs more frequently and thought their numbers had increased. ' +
                  'Cheetahs took the blame for most predator-related livestock losses. ' +
                  'Farmers killed cheetahs by the thousands as pests or to sell their skins ' +
                  'to the fur trade.',

                paragraph_Where_Did_The_Cheetah_Go_03:
                  'By 1975, researches realized that the cheetah was in trouble. CITES ' +
                  '(Convention on International Trade of Endangered Species) placed the cheetah ' +
                  'on Appendix I, making international trade in live cheetah or cheetah products illegal. ' +
                  'Local laws supported CITES in many countries where cheetahs still lived. ' +
                  'Researchers began looking for ways to encourage the growth of cheetah populations ' +
                  'through land management practices.'
              }
            }
          }
        }
      }
    }
  }
};
