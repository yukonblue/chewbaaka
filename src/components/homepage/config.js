/**
 * homepage/config.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 28, 2020
 * Updated  : Aug 17, 2021
 */

export const config = {
  components: {
    landingPageGateway: {
      constants: {
        landingPageGatewayTitle: "Learn about cheetahs",
        landingPageGatewaySubtitle: "Explore the Cheetah Museum at Cheetah Conservation Fund"
      },
    }
  }
}
