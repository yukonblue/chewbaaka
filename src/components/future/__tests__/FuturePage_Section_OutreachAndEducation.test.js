/**
 * FuturePage_Section_OutreachAndEducation.test.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 14, 2020
 * Updated  : Aug 21, 2021
 */

import React from 'react'

import { render } from '@testing-library/react'

import { config } from '../config'

import renderer from 'react-test-renderer'

import FuturePageSectionOutreachAndEducation from '../FuturePage_Section_OutreachAndEducation'

test('renders FuturePageSectionOutreachAndEducation component', () => {
  render(
    <FuturePageSectionOutreachAndEducation
      config={config}
    />
  );
});

test('FuturePageSectionOutreachAndEducation component snapshot', () => {
  const tree = renderer
    .create(
      <FuturePageSectionOutreachAndEducation
        config={config}
      />
    ).toJSON();
  expect(tree).toMatchSnapshot();
});
