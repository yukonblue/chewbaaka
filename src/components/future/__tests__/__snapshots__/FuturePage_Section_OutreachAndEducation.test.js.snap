// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FuturePageSectionOutreachAndEducation component snapshot 1`] = `
<article
  className="ContentPageSectionTemplateOuterContainer"
>
  <h2
    className="ContentPageSectionHeadTitle"
    id="outreach-and-education"
  >
    Outreach and Education
  </h2>
  <p
    className="ContentPageHeadAndSectionIntroText"
  >
    Public education is a vital component of any successful conservation strategies. Environmental and management education are critical in areas where people coexist intimately with the land and wildlife. Cheetah Conservation Fund believes public education and outreach will help ensure a future for the species in the wild. CCF educates farmers, teachers, students, and the public about methods to conserve biodiversity, as well as champion the idea that everybody can be a custodian of nature and has something to contribute.
  </p>
  <section
    className="ContentPageSubsectionTemplateOuterContainer"
  >
    <h3
      className="ContentPageSubsectionTitle"
    >
      Field Research
    </h3>
    <div>
      <div
        className="ContentPageSubsectionTwoColumnContentTemplateOuterContainer"
      >
        <div
          className="ContentPageSubsectionTwoColumnContentTemplateInnerContainer"
        >
          <div
            className="ContentPageSubsectionTwoColumnContentTemplateColumnInnerContainer"
          >
            <div>
              <div>
                <h4
                  className="ContentPageSubsectionSubtitle"
                >
                  Veterinary Clinic
                </h4>
                <p
                  className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                >
                  The HASS Family Research Center houses CCF’s registered on-site veterinary clinic. The clinic allows CCF to provide medical care for cheetahs within the sanctuary, CCF Livestock Guarding Dogs and CCF’s livestock. In addition, all cheetahs that CCF studies receive a full biomedical work-up and health assessment.
                </p>
                <p
                  className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                >
                  Other clinical procedures include routine medical checks, dental care and when necessary, surgery. While working with these animals, CCF research staff can collect biological samples that are used to do health evaluation, reproduction studies and to determine the cheetah population’s genetic status and provide the basis of a disease surveillance system for cheetahs, these biological samples also aid in future research.
                </p>
                <p
                  className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                >
                  Wild cheetahs that come into the clinic are marked with a transponder ear tag and/or fitted with a radio or satellite tracking collar before release back into the wild. CCF works with the Namibian Ministry of Environment and Tourism and neighboring farmers to determine a suitable habitat for relocation if necessary, or release on the farm where the cheetah was caught.
                </p>
              </div>
            </div>
          </div>
          <div
            className="ContentPageSubsectionTwoColumnContentTemplateColumnInnerContainer"
          >
            <div>
              <div>
                <h4
                  className="ContentPageSubsectionSubtitle"
                >
                  Ecological Studies
                </h4>
                <p
                  className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                >
                  Satellite tracking collars provide important data for CCF research teams. Researchers can monitor the daily movement of the cheetah, track its hunting skills and monitor its general well-being in the wild. If also allows researchers to evaluate the cheetah’s habitat use and prey preference.
                </p>
                <p
                  className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                >
                  Satellite and VHS monitoring collars, as well as camera traps help determine animal home ranges, habitat preferences and seasonal use, territoriality and behaviors unique to individual cheetah populations.
                </p>
                <p
                  className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                >
                  CCF conducts vegetation studies that include monitoring growth patterns of bush within targeted study areas on CCF land. Information on vegetation is used to identify target areas for ecological management and help determine and monitor the manner with which bush encroachment affects the biodiversity of the land.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="VerticalCushionPadding"
      >
        <div>
          <a
            href="https://cheetah.org/about/what-we-do/research/"
          >
            <button
              className="ui blue icon button"
              onClick={[Function]}
            >
              Learn more about CCF's research work
              <i
                aria-hidden="true"
                className="file alternate outline icon"
                onClick={[Function]}
              />
            </button>
          </a>
        </div>
      </div>
    </div>
  </section>
  <section
    className="ContentPageSubsectionTemplateOuterContainer"
  >
    <h3
      className="ContentPageSubsectionTitle"
    >
      Internships and Volunteering
    </h3>
    <div>
      <div
        className="ContentPageSubsectionPartContainer"
      >
        <div
          className="Centered FullWidth"
          style={
            Object {
              "width": 1300,
            }
          }
        >
          <img
            alt="CCF Volunteering"
            className="FullWidth DisplayBlock RetainAspectRatio"
            height={433}
            src="CCF_GetInvolved_Volunteer_L-min.jpg"
            width={1300}
          />
        </div>
      </div>
      <div
        className="ContentPageSubsectionPartContainer"
      >
        <div
          className="ContentPageSubsectionTwoColumnContentTemplateOuterContainer"
        >
          <div
            className="ContentPageSubsectionTwoColumnContentTemplateInnerContainer"
          >
            <div
              className="ContentPageSubsectionTwoColumnContentTemplateColumnInnerContainer"
            >
              <div>
                <div>
                  <h4
                    className="ContentPageSubsectionSubtitle"
                  >
                    Internships
                  </h4>
                  <p
                    className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                  >
                    CCF welcomes a selected number of students each year with a wide variety of backgrounds and courses of study. Graduate and undergraduate students are given a chance to learn about aspects of conservation biology, ecology, veterinary medicine, genetics, animal science, agriculture, sociology, business, computing, graphic design, education and tourism.
                  </p>
                  <p
                    className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                  >
                    CCF is an official fourth year placement for students from Namibia’s two major universities, University of Namibia (UNAM) and Namibia University of Science and Technology (NUST). In addition, CCF welcomes interns from undergraduate and graduate university programs all over the world.
                  </p>
                  <p
                    className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                  >
                    Aspiring biologists, genetics, and ecologists pursuing Masters and Ph.D. degrees come to CCF to work on research and thesis projects all year.
                  </p>
                  <ul>
                    <li
                      className="ContentPageRegularTextSize ContentPageRegularTextLineHeight"
                    >
                      cheetah feeding - collecting and dissecting cheetah feces
                    </li>
                    <li
                      className="ContentPageRegularTextSize ContentPageRegularTextLineHeight"
                    >
                      observing cheetah behavior - assisting in clinical work-ups
                    </li>
                    <li
                      className="ContentPageRegularTextSize ContentPageRegularTextLineHeight"
                    >
                      raise livestock guarding dog puppies - conducting field work
                    </li>
                    <li
                      className="ContentPageRegularTextSize ContentPageRegularTextLineHeight"
                    >
                      being involved with outreach and education programs
                    </li>
                  </ul>
                </div>
              </div>
            </div>
            <div
              className="ContentPageSubsectionTwoColumnContentTemplateColumnInnerContainer"
            >
              <div>
                <div>
                  <h4
                    className="ContentPageSubsectionSubtitle"
                  >
                    Volunteering
                  </h4>
                  <p
                    className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                  >
                    Volunteering at CCF in Namibia is a rewarding experience, and most of the people who participate in our volunteer program will tell you that it is among the best experiences they have ever had. Volunteers do everything from data entry and camera trap photo sorting to chopping up meat to collecting and cataloging scat samples.
                  </p>
                  <p
                    className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                  >
                    Like interns, volunteers can expect to work on a variety of tasks. CCF also seeks professionally skilled volunteers to work as cheetah keepers, animal behavior specialists, ecologists, biologists, veterinarians, and vet-technicians, as well as educators, trainers, and conservationists.
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="ContentPageSubsectionPartContainer"
      >
        <div>
          <a
            href="https://cheetah.org/get-involved/volunteer/"
          >
            <button
              className="ui blue icon button"
              onClick={[Function]}
            >
              Learn more about CCF's Volunteer Program
              <i
                aria-hidden="true"
                className="file alternate outline icon"
                onClick={[Function]}
              />
            </button>
          </a>
        </div>
      </div>
    </div>
  </section>
  <section
    className="ContentPageSubsectionTemplateOuterContainer"
  >
    <h3
      className="ContentPageSubsectionTitle"
    >
      The Role of Zoos
    </h3>
    <div>
      <div
        className="ContentPageSubsectionPartContainer"
      >
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          Moderns zoos serve as conservation education centers and provide sanctuary for many endangered species. Cooperative breeding, management and research have assisted in re-establishing some species in the wild.
        </p>
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          Historically, cheetahs have not bred well in captivity. In recent years, through cooperative research and management in zoos worldwide, a global master plan is producing successful results. If future extinction were to occur with wild cheetahs, this research and maintained captive populations could allow the re-introduction of cheetahs.
        </p>
      </div>
      <div
        className="ContentPageSubsectionPartContainer"
      >
        <h4
          className="ContentPageSubsectionSubtitle"
        >
          Ambassadors
        </h4>
        <div
          className="ContentPageSideFloatFluidContainerOuterContainer"
        >
          <div
            className="ContentPageSideFloatFluidContainerFloatPartContainer BottomMargin20px FloatRight"
          >
            <div
              className="ImageViewOuterContainer"
              style={
                Object {
                  "maxWidth": 720,
                }
              }
            >
              <div
                className="ImageViewInnerContainer ImageViewFamilyContainerBorderStyles ImageViewFamilyContainerShadow ImageViewFamilyContainerBackgroundStyles"
              >
                <div
                  className="ImageViewImgContainer"
                >
                  <img
                    alt="In 2009, Cincinnati Zoo's cheetah Sarah set the record of the world's fastest cheetah, as she sprant 100m in just 6.13 seconds. The best record for 100m by humans was from Usain Bolt, finishing in 9.58 seconds."
                    className="ImageViewFamilyImgLayoutStyles ImageViewFamilyImgBorderStyles"
                    height={406}
                    src="cincinnati_zoo_cheetah_sarah-min.jpg"
                    width={720}
                  />
                </div>
                <div
                  className="ImageViewFamilyCaptionContainerStyles"
                >
                  <div
                    className="CaptionContainer"
                    data-testid="CaptionComponentTestId"
                    style={
                      Object {
                        "maxWidth": 720,
                      }
                    }
                  >
                    <div
                      className="IconContainer"
                    >
                      <i
                        aria-hidden="true"
                        className="image icon"
                        onClick={[Function]}
                      />
                    </div>
                    <p
                      className="ContentPageCaptionTextSize CaptionText"
                    >
                      In 2009, Cincinnati Zoo's cheetah Sarah set the record of the world's fastest cheetah, as she sprant 100m in just 6.13 seconds. The best record for 100m by humans was from Usain Bolt, finishing in 9.58 seconds.
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            className="ContentPageSideFloatFluidContainerFixedPartContainer"
          >
            <p
              className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
            >
              As representatives of the wild cheetah, zoo animals provide the opportunity for people around the world to see a cheetah close up.
            </p>
            <p
              className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
            >
              Through educational programs, zoos spark the interest of individuals who can support or join the projects that will save wild cheetahs.
            </p>
            <p
              className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
            >
              Zoos contribute to the conservation of wild cheetahs through education, research and support.
            </p>
          </div>
        </div>
      </div>
      <div
        className="ContentPageSubsectionPartContainer"
      >
        <h4
          className="ContentPageSubsectionSubtitle"
        >
          Research
        </h4>
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          Zoos, universities, private individuals, governmental and NGOs support cooperative efforts worldwide.
        </p>
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          Reproductive, medical and behavioral studies, and laboratory analysis are cooperative conservation efforts between captive and wild cheetah.
        </p>
      </div>
    </div>
  </section>
  <section
    className="ContentPageSubsectionTemplateOuterContainer"
  >
    <h3
      className="ContentPageSubsectionTitle"
    >
      Schools, Teachers, Learners
    </h3>
    <div>
      <div
        className="ContentPageSubsectionTwoColumnContentTemplateOuterContainer"
      >
        <div
          className="ContentPageSubsectionTwoColumnContentTemplateInnerContainer"
        >
          <div
            className="ContentPageSubsectionTwoColumnContentTemplateColumnInnerContainer"
          >
            <div>
              <p
                className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
              >
                CCF develops outreach school programs and educational packets are distributed to students and teachers. All schools are welcomed to CCF facilities.
              </p>
              <p
                className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
              >
                Staff members conduct assemblies throughout Namibia. Students are encouraged to become more involved in conservation in school and in their homes. Workshops assist teachers in initiating cheetah conservation and environmental education programs.
              </p>
              <p
                className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
              >
                Educational facilities at CCF’s headquarters encourage participation in cheetah conservation. The Education Center, the Visitor Center and Predator-Preyground provide information in an interacting setting. CCF’s Wildlife Camp allows outdoor education and a wilderness experience. A Nature Trail highlights the farmland ecosystem and the valuable role of predators.
              </p>
              <p
                className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
              >
                CCF takes a cross-curricular approach, integrating conservation issues into subjects that are required as part of the school syllabus and donates books to schools and libraries.
              </p>
              <p
                className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
              >
                CCF works with government agencies to incorporate cheetah conservation in the school curriculum.
              </p>
            </div>
          </div>
          <div
            className="ContentPageSubsectionTwoColumnContentTemplateColumnInnerContainer"
          >
            <div>
              <div
                className="ImageSlidingGalleryOuterContainer"
                style={
                  Object {
                    "maxWidth": undefined,
                  }
                }
              >
                <div
                  className="ImageSlidingGalleryCoreContainer"
                >
                  <div
                    className="ImageViewFamilyContainerBorderStyles ImageViewFamilyContainerShadow ImageViewFamilyContainerBackgroundStyles"
                  >
                    <img
                      alt="CCF's Predator Preyground."
                      className="ImageViewFamilyImgLayoutStyles ImageViewFamilyImgBorderStyles"
                      height={320}
                      src="CCF_Predator_Preyground-min.jpg"
                      width={533}
                    />
                    <div
                      className="ImageViewFamilyCaptionContainerStyles"
                      data-testid="ImageSlidingGalleryCaptionPart"
                    >
                      <div
                        className="CaptionContainer"
                        data-testid="CaptionComponentTestId"
                        style={
                          Object {
                            "maxWidth": undefined,
                          }
                        }
                      >
                        <div
                          className="IconContainer"
                        >
                          <i
                            aria-hidden="true"
                            className="image icon"
                            onClick={[Function]}
                          />
                        </div>
                        <p
                          className="ContentPageCaptionTextSize CaptionText"
                        >
                          CCF's Predator Preyground.
                        </p>
                      </div>
                    </div>
                  </div>
                  <div
                    className="ImageSlidingGalleryButtonContainer"
                  >
                    <div
                      aria-label="Previous slide"
                      className="ui blue huge icon left attached disabled button"
                      onClick={[Function]}
                      role="button"
                      tabIndex={-1}
                    >
                      <i
                        aria-hidden="true"
                        className="arrow alternate circle left outline icon"
                        onClick={[Function]}
                      />
                    </div>
                    <div
                      aria-label="Next slide"
                      className="ui blue huge icon right attached button"
                      onClick={[Function]}
                      role="button"
                      tabIndex={0}
                    >
                      <i
                        aria-hidden="true"
                        className="arrow alternate circle right outline icon"
                        onClick={[Function]}
                      />
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
  <div>
    <section
      className="ContentPageSubsectionTemplateOuterContainer"
    >
      <h3
        className="ContentPageSubsectionTitle"
      >
        Community Events
      </h3>
      <div
        className="ContentPageSideFloatFluidContainerOuterContainer"
      >
        <div
          className="ContentPageSideFloatFluidContainerFloatPartContainer BottomMargin20px FloatLeft"
        >
          <div
            className="ImageViewOuterContainer"
            style={
              Object {
                "maxWidth": 960,
              }
            }
          >
            <div
              className="ImageViewInnerContainer ImageViewFamilyContainerBorderStyles ImageViewFamilyContainerShadow ImageViewFamilyContainerBackgroundStyles"
            >
              <div
                className="ImageViewImgContainer"
              >
                <img
                  alt="CCF helps Namibian artists and artisans to market and sell their work with the Livelihood Development Program."
                  className="ImageViewFamilyImgLayoutStyles ImageViewFamilyImgBorderStyles"
                  height={577}
                  src="CCF_Livelihood_Development_L-min.jpg"
                  width={960}
                />
              </div>
              <div
                className="ImageViewFamilyCaptionContainerStyles"
              >
                <div
                  className="CaptionContainer"
                  data-testid="CaptionComponentTestId"
                  style={
                    Object {
                      "maxWidth": 960,
                    }
                  }
                >
                  <div
                    className="IconContainer"
                  >
                    <i
                      aria-hidden="true"
                      className="image icon"
                      onClick={[Function]}
                    />
                  </div>
                  <p
                    className="ContentPageCaptionTextSize CaptionText"
                  >
                    CCF helps Namibian artists and artisans to market and sell their work with the Livelihood Development Program. (Image credit: Cheetah Conservation Fund)
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="ContentPageSideFloatFluidContainerFixedPartContainer"
        >
          <p
            className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
          >
            Participatory community events, such as competitions in art, poetry and writing, increase conservation awareness.
          </p>
          <p
            className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
          >
            Public educational displays are maintained at strategic locations and community events. Displays provide introductory information on the plight of the cheetah and the work conducted by CCF.
          </p>
          <p
            className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
          >
            CCF staff participate in farmer meetings, local conservancy activities and other related events. Workshops and seminars are conducted to exchange information and develop conservation-based farming practices.
          </p>
        </div>
      </div>
    </section>
  </div>
</article>
`;
